### ISSUE TITLE
TransformableInfo Typing out of date.

### ISSUE THREAD

**Issue by dscalzi** [winstonjs/logform#111]
The `TransformableInfo` key is no longer restricted to only a string. `triple-beam` changed their typings such that the constants are no longer strings.

see triple-beam change: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/46490

usage in winston: https://github.com/winstonjs/winston/blob/68ea786e053e50b51396e0cc8993c67ff789609c/lib/winston/logger.js#L251

Location of required change: https://github.com/winstonjs/logform/blob/master/index.d.ts#L10

TL;DR the type needs to support strings and `unique symbol`.

Current ugly workaround for projects using these typings:
```typescript
import { SPLAT as SPLAT_Symbol } from 'triple-beam'
const SPLAT = SPLAT_Symbol as unknown as string // this is not a string

...
format.printf(info => {
    if(info[SPLAT]) { ... }
})
```

**Comment by bradennapier** [comment]
Yep, running into the same

**Comment by joechrz** [comment]
Any plans to merge this PR?  Running into this issue as well.

**Comment by dscalzi** [comment]
@yurijmikhalevich can this be given some attention? Still a problem over a year later and it's causing a big headache getting tsc to work on gh-actions.

**Comment by dscalzi** [comment]
Project seems abandoned. Shame.
